kview/src/main/java/dev/ikm/komet/kview/mvvm/viewmodel/StampCreateFormViewModel.java

```diff
@@ -0,0 +1,149 @@
+package dev.ikm.komet.kview.mvvm.viewmodel;
+
+import dev.ikm.komet.framework.view.ViewProperties;
+import dev.ikm.komet.kview.events.ClosePropertiesPanelEvent;
+import dev.ikm.komet.kview.mvvm.view.genediting.ConfirmationDialogController;
+import dev.ikm.tinkar.composer.Composer;
+import dev.ikm.tinkar.composer.Session;
+import dev.ikm.tinkar.entity.*;
+import dev.ikm.tinkar.events.EvtBusFactory;
+import dev.ikm.tinkar.events.Subscriber;
+import dev.ikm.tinkar.terms.*;
+import javafx.event.ActionEvent;
+import javafx.scene.Node;
+import org.carlfx.cognitive.validator.ValidationResult;
+import org.carlfx.cognitive.viewmodel.ViewModel;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+
+import static dev.ikm.komet.kview.mvvm.model.DataModelHelper.fetchDescendentsOfConcept;
+import static dev.ikm.komet.kview.mvvm.viewmodel.StampFormViewModelBase.StampProperties.*;
+
+public class StampCreateFormViewModel extends StampFormViewModelBase {
+    /**
+     * Provide the standard Confirm Clear dialog title for use in other classes
+     */
+    public static final String CONFIRM_RESET_TITLE = "Confirm Clear Form";
+    /**
+     * Provide the standard Confirm Clear dialog message for use in other classes
+     */
+    public static final String CONFIRM_RESET_MESSAGE =  "Are you sure you want to clear the form? All entered data will be lost.";
+
+    private EntityFacade entityFacade;
+    private UUID topic;
+
+    private Subscriber<ClosePropertiesPanelEvent> closePropertiesPanelEventSubscriber;
+
+    public StampCreateFormViewModel(StampType stampType) {
+        super(stampType);
+        // Add Properties
+        addProperty(STATUS, (State) null);
+        addProperty(TIME, 0L);
+        addProperty(AUTHOR, (ComponentWithNid) null);
+        addProperty(MODULE, (ComponentWithNid) null);
+        addProperty(PATH, (ComponentWithNid) null);
```

Comment:
Doesn't the super class contain these properties? If not they should.

Reply:
@carldea good catch! There is indeed duplication of code here that can be moved to the super class. I missed that.